server/test/integration/serverTest.js

```diff
@@ -11,7 +11,7 @@ const { ALERT_STATE, helpers } = require('brave-alert-lib')
 chai.use(chaiHttp)
```

Comment:
Eventually, I'd like to split this file up into a bunch of little ones too, but that's further away from what the changes that I was making for the API key, so I decided that it can wait until there are more `server.js` changes (or until the `server.js` refactor Task) to do this.